modules/swagger-codegen/src/main/resources/swift3/Models.mustache

```diff
@@ -10,8 +10,10 @@ protocol JSONEncodable {
     func encodeToJSON() -> Any
 }
 
-public enum ErrorResponse : Error {
-    case Error(Int, Data?, Error)
+public struct ErrorResponse : Error {
+    public let statusCode: Int
+    public let data: Data?
+    public let error: Error
 }
```

Comment:
Would also be lovely to implement `CustomStringConvertible` and print out the response data as a string.